README.md

```diff
@@ -176,7 +178,7 @@ This command returns the something similar to this:
 sudo ./run.sh shm_size 12G
 ```
 
-1. Open the config file in your favourite text editor. This example shows how to do it using Nano, an editor that is good for beginners.
+1. Open the config file in your favorite text editor. This example shows how to do it using Nano, an editor that is good for beginners.
```

Comment:
"favourite" was already correct, it is the British English version, while "favorite" is the americanized version.

Reply:
Agreed.